src/module.js

```diff
@@ -117,7 +117,7 @@ _.extend(Marionette.Module.prototype, Backbone.Events, {
       Marionette,
       Backbone.$, _,
       customArgs
-    ]);
+    ], true);
```

Comment:
It's tricky because in lodash its `flatten(collection, deep)` and in underscore its `flatten(collection, shallow)`